app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/email/RegistrationEnterEmailController.scala

```diff
@@ -31,9 +32,10 @@ import uk.gov.hmrc.cgtpropertydisposalsfrontend.repos.SessionStore
 import uk.gov.hmrc.cgtpropertydisposalsfrontend.services.EmailVerificationService
 import uk.gov.hmrc.cgtpropertydisposalsfrontend.util.Logging
 import uk.gov.hmrc.cgtpropertydisposalsfrontend.{controllers, views}
+import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
-
-import scala.concurrent.ExecutionContext
+import cats.implicits._
```

Comment:
again, I think you should be able to just import `cats.instances.future._`

Reply:
fixed